exercises/bankaccount.js

```diff
@@ -2,17 +2,38 @@
 
 class Account{
```

Comment:
Like the bank object, the Account object has certain methods that it needs to perform its job. In this instance, we have methods that add money to an account, remove money from an account, and return the total amount of money in an account.Take a moment and consider what other methods you might want to add in you were to build this object out further. Thought experiments like this are good for practicing the conceptualization of objects.